packages/tokens/config.js

```diff
@@ -14,15 +14,32 @@ module.exports = {
       prefix: '',
       files: [
         {
-          destination: 'js/index.js',
+          destination: 'js/colors.js',
           format: 'javascript/es6/vads-colors',
           filter: 'filter/color/is-color',
         },
         {
-          destination: 'index.d.ts',
+          destination: 'js/themes.js',
+          format: 'javascript/es6/vads-colors-themes',
+        },
+        {
+          destination: 'js/index.js',
+          format: 'javascript/es6/vads-module-export',
+        },
+        {
+          destination: 'types/colors.d.ts',
           format: 'typescript/es6-declarations/colors',
           filter: 'filter/color/is-color',
         },
+        {
+          destination: 'types/theme.d.ts',
+          format: 'typescript/es6-declarations/theme',
+          filter: 'filter/color/light-mode',
+        },
+        {
+          destination: 'index.d.ts',
```

Comment:
Perhaps should shift this to the top so the list is alphabetical as this list will continue growing with other token sets.

Reply:
Sorted all these by `destination`